issues-policy.md

```diff
@@ -0,0 +1,84 @@
+# GitHub issues process and policy
+
+The goals of the process are:
+
+1. Ensure errors or omissions in our docs are not blocking customer success.
+1. Be responsive to customer feedback and concerns.
+1. Continually improve the customer experience.
+1. Learn more about customer experiences through dialogue in the open on challenges and solutions.
+
+The process uses two stages to ensure responsiveness while prioritizing work. The initial stage diagnoses and triages the issue. The second stage resolves the issue. When an issue is both easy and urgent, the two stages may be combined.
+
+The process involves two time boxed tasks:
+
+- Each team member spends up to 1/2 hour each business day [classifying incoming issues](#diagnosis-phase), including initial response. This ensures we are responsive on new issues.
+- Each team member spends up to 1.5 days per sprint [updating documents](#resolution-phase) fixing customer generated GitHub issues.
+
+## Diagnosis phase
+
+Every team member spends up to 30 minutes per business day categorizing new issues. We answer the following questions:
+
+- Is the issue a docs issue, or a product issue?
+- Is it not an issue, but a question better suited for a forum or support site?
+- What priority is the issue?
+- What area manages this issue?
+
+If these questions can't be answered in the initial examination, we ask clarifying questions in the comments.
+
+There are types of issues that are closed during the diagnosis and triage phase:
+
+- **kudos**: Express thanks, and close the issue.
+- **product issue**: Issues that request product changes are closed. We may also take additional actions, as described below.
```

Comment:
product changes only? Perhaps, "Issues that are related to the product and not to its documentation"?